Source/WebCore/page/scrolling/mac/ScrollerMac.mm

```diff
@@ -369,7 +369,7 @@ - (void)invalidate
 
 void ScrollerMac::updateScrollbarStyle()
 {
-    m_scrollerImp = [NSScrollerImp scrollerImpWithStyle:nsScrollerStyle(m_pair.scrollbarStyle()) controlSize:NSControlSizeRegular horizontal:m_orientation == ScrollbarOrientation::Horizontal replacingScrollerImp:takeScrollerImp().get()];
+    m_scrollerImp = [NSScrollerImp scrollerImpWithStyle:nsScrollerStyle(m_pair.scrollbarStyle()) controlSize:nsControlSizeFromScrollbarWidth(m_pair.scrollbarWidthStyle()) horizontal:m_orientation == ScrollbarOrientation::Horizontal replacingScrollerImp:takeScrollerImp().get()];
```

Comment:
Setting replacingScrollerImp to nil here fixes the specific rendering issue with non UISideCompositing but obviously breaks the code.

Not sure if this information is useful to working out why it breaks.